rh-tictactoe/src/app/screens/Game/index.js

```diff
@@ -4,34 +4,23 @@ import { connect } from 'react-redux';
 
 import { actionCreator } from '../../../../src/redux/game/actions.js';
 
-import Styles from './styles.scss';
-import Board from './components/Board';
-import Moves from './components/Moves';
+import Game from './layout.js';
 
-class Game extends React.Component {
-  render() {
-    return (
-      <div className={Styles.game}>
-        <div className={Styles.gameBoard}>
-          <Board
-            squares={this.props.history[this.props.stepNumber].squares}
-            onClick={this.props.handleClick}
-            status={this.props.status}
-            history={this.props.history}
-          />
-        </div>
-        <div className={Styles.gameInfo}>
-          <div>{this.props.status}</div>
-          <Moves onClick={this.props.jumpTo} moves={this.props.history} />
-        </div>
-      </div>
-    );
-  }
+function GameContainer(props) {
+  return (
+    <Game
+      history={props.history}
+      jumpTo={props.jumpTo}
+      onClick={props.handleClick}
+      status={props.status}
+      stepNumber={props.stepNumber}
+    />
+  );
 }
 
-Game.propTypes = {
+GameContainer.propTypes = {
   handleClick: PropTypes.func.isRequired,
-  history: PropTypes.arrayOf(PropTypes.object),
+  history: PropTypes.arrayOf(PropTypes.shape({ squares: PropTypes.array })),
```

Comment:
array of what? We always are specific to know where the problem can be in the future and make the code better